### PR TITLE
docs: add catppuccin/zen-browser

### PR DESCRIPTION
**Pull request by IAmJafeth**
Closes https://github.com/catppuccin/catppuccin/issues/2544

### CODE REVIEW

File: resources/ports.yml [hunk 1]
```diff
@@ -2222,6 +2224,13 @@ ports:
     color: blue
     current-maintainers: [*tecandrew]
     icon: zedindustries
+  -browser:
```
Comment (uncenter) [inline resources/ports.yml:2227]:
```suggestion
  zen-browser:
```

File: resources/ports.yml [hunk 2]
```diff
@@ -2222,6 +2224,13 @@ ports:
...
+    name: Zen Browser
+    categories: [browser]
+    platform: [macos, linux, windows]
+    color: sapphire
+    current-maintainers: [*IAmJafeth]
+    icon: zen-browser
```
Comment (uncenter) [inline resources/ports.yml:2233]:
```suggestion
```
There is no icon for this brand yet on https://simpleicons.org/ where these are sourced from.

File: resources/ports.yml [hunk 3]
```diff
@@ -2222,6 +2224,13 @@ ports:
     color: blue
     current-maintainers: [*tecandrew]
     icon: zedindustries
+  -browser:
+    name: Zen Browser
+    categories: [browser]
+    platform: [macos, linux, windows]
+    color: sapphire
```
Comment (uncenter) [inline resources/ports.yml:2231]: They seem to pretty much use crust/text in terms of branding colors 🤔

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by sgoudham — APPROVED** [review]
Thanks!

**Pull request merged**